elasticsearch/src/main/java/org/hibernate/search/elasticsearch/analyzer/definition/ElasticsearchTypedAnalysisComponentDefinitionContext.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.elasticsearch.analyzer.definition;
+
+
+/**
+ * @author Yoann Rodiere
+ */
+public interface ElasticsearchTypedAnalysisComponentDefinitionContext {
+
+	ElasticsearchTypedAnalysisComponentDefinitionContext param(String name, String value);
+
+	ElasticsearchTypedAnalysisComponentDefinitionContext param(String name, String... values);
```

Comment:
I was wondering why the String parameter has a vararg option, and the others don't. I guess you have a specific analyzer need example in mind?